modules/claims_api/spec/sidekiq/report_hourly_unsuccessful_submissions_spec.rb

```diff
@@ -70,7 +70,7 @@
                                                                                  transaction_id: 'transaction_3',
                                                                                  id: '4')
         expected_array = [claim_three.id,
-                          claim_four.id] || [claim_four.id, claim_three.id]
+                          claim_four.id] || [claim_four.id, claim_three.id].count
         # rubocop:disable RSpec/SubjectStub
```

Comment:
@stiehlrod  I believe if the right side of that was checked it would error out because '2' is not received by the job. 

I think the only way to make this behave reliably  is to take out the expectation of the  ordering.  Something like this appears to work

```
        expected_present_values = [
          [], 
          [claim_three.id, claim_four.id],
          ['poa1'],
          ['itf1'],
          ['ews1']
        ]
        # rubocop:disable RSpec/SubjectStub
        expect(subject).to receive(:notify) do |*args|
          args.each_with_index do |arg, idx|
            if [5,6,7].include?(idx)
              expect(arg).to be_a(String) # these are the date/times that get sent
            else
              expect(expected_present_values).to include(arg)
            end
          end
        end
```